catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -21,26 +21,48 @@
            http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
            http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
 
+    <reference-list id="inputTransformers" interface="ddf.catalog.transform.InputTransformer"
+                    member-type="service-reference" filter="(id=*)"/>
 
     <reference id="camelContext" interface="org.apache.camel.CamelContext"
                filter="(camel.context.name=catalogCamelContext)"/>
 
     <reference id="attributeRegistry" interface="ddf.catalog.data.AttributeRegistry"/>
 
+    <bean id="executorService" class="java.util.concurrent.Executors"
+          factory-method="newSingleThreadExecutor">
+        <argument ref="cdmThreadFactoryBuilder"/>
+    </bean>
+
+    <bean id="cdmThreadFactoryBuilder"
+          class="org.codice.ddf.platform.util.StandardThreadFactoryBuilder"
+          factory-method="newThreadFactory">
+        <argument value="contentDirectoryMonitorThread"/>
+    </bean>
+
+    <bean id="inputTransformerIds" class="org.codice.ddf.catalog.content.monitor.InputTransformerIds"/>
```

Comment:
✏️This can be inlined
```xml
<argument ref="executorService"/>
<argument>
    <bean class="org.codice.ddf.catalog.content.monitor.InputTransformerIds"/>
</argument>
<argument value="30000"/>
```